doc/development.md

```diff
@@ -49,11 +49,51 @@ To get started quickly, it is recommended to use Gitpod as default with the deve
   - [https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers](https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers)
   - [https://www.youtube.com/watch?v=idW-an99TAM](https://www.youtube.com/watch?v=idW-an99TAM)
 
-- Instal Elasticsearch with Docker
+- Install Elasticsearch with Docker
 
   - sudo docker pull elasticsearch:7.9.3
   - sudo docker run -d -name elasticsearch -p 9200:9200 -p 9300:9300 -e “discovery.type=single-node” elasticsearch:7.9.3
 
+### Setup locally on MacOS
+
+- Set up postgresql using homebrew
+
+  - brew install postgresql@12
+
+  - Use "postgres –V" to check if it is the correct version
+
+  - brew services start postgresql@12
+
+  - From the terminal, run the command "createdb postgres" (NOT inside psql)
+
+  - "psql –d postgres" to enter the database
+
+  - CREATE ROLE gitpod with LOGIN PASSWORD 'gitpod';
+
+  - Using \d to display all tables and relations, now empty
+
+- Make sure using the correct java version
+
+  -Download java 11 if not yet downloaded yet
+
+  - Run the command "/usr/libexec/java_home –V" to see your matching java virtual machines
```

Comment:
question: what does "matching virtual machines" here mean? Specifically, the "matching" part

```suggestion
  - Run the command "/usr/libexec/java_home –V" to see your matching Java virtual machines
```